src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random;
+
+    public ColorSupplier() {
+        this.random = new Random();
+    }
+
+    public String getRandomColor() {
+        return String.valueOf(Color.values()[random.nextInt(Color.values().length)]);
```

Comment:
Use name() for getting String representation of enum constants. Don't use `toString()` or `valueOf()`.
```suggestion
        int index = random.nextInt(Color.values().length);
        return Color.values()[index].name();
```